retrieval/scrape.go

```diff
@@ -297,6 +307,7 @@ func (s *targetScraper) scrape(ctx context.Context, ts time.Time) (model.Samples
 	}
 	req.Header.Add("Accept", acceptHeader)
 	req.Header.Set("User-Agent", userAgentHeader)
+	req.Header.Set("Scrape-Timeout-Seconds", fmt.Sprintf("%f", s.timeout.Seconds()))
```

Comment:
You could make the same argument for any of the other protocols I mentioned above, so I think adding the `Prometheus` in there still makes sense, since we define its meaning.